src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs

```diff
@@ -70,11 +70,35 @@ public UInt256 GetGasPriceEstimate()
             LastGasPrice = gasPriceEstimate;
             return gasPriceEstimate!;
         }
+        
+        public UInt256 GetGasPriceEstimateWithBaseFee()
+        {
+            Block? headBlock = _blockFinder.Head;
+            if (headBlock is null)
+            {
+                return FallbackGasPrice();
+            }
+            
+            if (LastGasPrice is not null && LastHeadBlock!.Hash == headBlock!.Hash)
+            {
+                return LastGasPrice.Value;
```

Comment:
Is it safe to rely on same value as GetGasPriceEstimate? 
If we call GetGasPriceEstimate first it will set this value.
Shouldn't it be a separate value just for this method?